### ISSUE TITLE
[Feature] There is an error in the official tutorial

### ISSUE THREAD

**Issue by BiFangKNT** [MauriceNino/dashdot#945]
### Description of the feature

In the docker tutorial of the installation options in the documentation,The port mapping is `80:3001`. However, in docker, the port before the colon is the external port, and the latter is the internal port. If installed according to the official tutorial, port 3001 of the container will be mapped to port 80 of the host. 

After execution, an error will be returned: `Error starting userland proxy: listen tcp4 0.0.0.0:80: listen: address already in use.`, which is very strange. 

This is obviously wrong,It should be changed to `3001:80`.

**Comment by MauriceNino** [comment]
@defconxt - I think there is a misunderstanding on how Docker works. The port mapping just tells docker where to route to. There is no (good) way to read this from the application, that's why changing the internal port will not change the port that dash. listens to. This is by design, and I can't think of any other Docker container that would do that for you.

If you want to change the internal port (and I doubt that you really want to do that), you can use the configuration variable [DASHDOT_PORT](https://getdashdot.com/docs/config/features#dashdot_port). This will not solve any of your port conflicts though, as inside the container there is only one process running on one port anyways. You need to fix the internal port - e.g. `8922:3001`.

**Comment by BiFangKNT** [comment]
> 不幸的是，由于无法更改端口，我也无法使用此应用程序，文档也非常具有误导性。
> 
> 请参阅： https: [//github.com/MauriceNino/dashdot](https://github.com/MauriceNino/dashdot) 和 https://getdashdot.com/docs/install/docker-compose
> 
> 它仍然显示设置 80:3001，将配置切换到 8922:80 不起作用，它会忽略我的请求，而是在我以正确的顺序交换端口后成功启动并侦听 3001。我在 3001 上运行着其他服务。
> 
> ```
> 2023-12-12T01:42:49.413014294Z yarn run v1.22.19
> 2023-12-12T01:42:49.444022840Z $ node .
> 2023-12-12T01:42:49.568110318Z listening on *:3001
> ```
> 
> 期望：允许我设置自己的主机端口，而不是被迫使用 3001
> 
> 结果：不能。

Maybe you misunderstood the corresponding relationship between port settings. The left side of the colon is the host port, and the right side is the container internal port. The container port defaults to 3001, which does not affect other ports you map to the outside. For example, if port 3010 in your host port is free, you can set it like this:`3010:3001`.

The reason why the document is written like this may be just that it is a schematic demonstration and has no actual guiding significance. Maybe they think that users who use this program all have certain knowledge about ports and do not need to specifically explain this, but for those who are not familiar with it, For users of the docker command, it is indeed misleading.

**Comment by MauriceNino** [comment]
> Maybe they think that users who use this program all have certain knowledge about ports and do not need to specifically explain this

Yes that's the exact reason tbh. If you have any recommendations on what to add to the docs, to make it clearer to people without that knowledge, please let me know - or you can propose a change via the `Edit this page` button on the bottom of every page in the docs.

![image](https://github.com/MauriceNino/dashdot/assets/18658092/754466bc-b6b6-4daa-8d99-4d8a65b94e73)

**Comment by BiFangKNT** [comment]
> > 也许他们认为使用这个程序的用户都对端口有一定的了解，不需要专门解释这一点
> 
> 是的，这就是确切的原因。如果您对添加到文档中的内容有任何建议，为了让不了解这些知识的人更清楚，请告诉我 - 或者您可以通过`Edit this page`文档中每个页面底部的按钮提出更改。
> 
> ![图像](https://private-user-images.githubusercontent.com/18658092/289927395-754466bc-b6b6-4daa-8d99-4d8a65b94e73.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI0MDMxNDAsIm5iZiI6MTcwMjQwMjg0MCwicGF0aCI6Ii8xODY1ODA5Mi8yODk5MjczOTUtNzU0NDY2YmMtYjZiNi00ZGFhLThkOTktNGQ4YTY1Yjk0ZTczLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjEyVDE3NDA0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ4ZjE4MzA1MDhiNWYwMWI0Y2RiMzZhM2RkYzE5NzA4YTQ1ODY0NGE5ZTZhNTg1MWRjMWE4OTIxY2M0ZGE2ZTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.0m5XCPT7wTcqlp-1airah-qja8u42rvv2x1itY5hE2E)

If you hope that users who don’t understand docker commands can correctly understand the use of your program without ambiguity, I think you can change the example to: `3001:3001`, and indicate: 

> The left side of the colon is the host port , the one on the right is the container's internal port. The container's internal port defaults to 3001, and the host port can be customized according to your actual needs.

As for why it is not recommended to set the host port in the example to 80, it is because in mainland China, ports 80 and 443 of civilian broadband IP are generally blocked by operators. Opening these two ports requires filing first, and ordinary Users generally do not go through this procedure.
